apps/docs/components/HomePageCover.tsx

```diff
@@ -30,76 +73,18 @@ const HomePageCover = (props) => {
           </p>
         </div>
         <div className="flex flex-wrap md:grid md:grid-cols-4 2xl:grid-cols-7 gap-2 sm:gap-3">
-          <Link href={`/guides/getting-started/quickstarts/reactjs`} passHref>
-            <a className="no-underline">
-              <IconPanel
-                iconSize={iconSize}
-                hideArrow
-                tooltip="ReactJS"
-                icon="/docs/img/icons/react-icon"
-              />
-            </a>
-          </Link>
-          <Link href={`/guides/getting-started/quickstarts/nextjs`} passHref>
-            <a className="no-underline">
-              <IconPanel
-                iconSize={iconSize}
-                hideArrow
-                tooltip="NextJS"
-                icon="/docs/img/icons/nextjs-icon"
-              />
-            </a>
-          </Link>
-          <Link href={`/guides/getting-started/quickstarts/redwoodjs`} passHref>
-            <a className="no-underline">
-              <IconPanel
-                iconSize={iconSize}
-                hideArrow
-                tooltip="RedwoodJS"
-                icon="/docs/img/icons/redwoodjs-icon"
-              />
-            </a>
-          </Link>
-          <Link href={`/guides/getting-started/quickstarts/flutter`} passHref>
-            <a className="no-underline">
-              <IconPanel
-                iconSize={iconSize}
-                hideArrow
-                tooltip="Flutter"
-                icon="/docs/img/icons/flutter-icon"
-              />
-            </a>
-          </Link>
-          <Link href={`/guides/getting-started/quickstarts/sveltekit`} passHref>
-            <a className="no-underline">
-              <IconPanel
-                iconSize={iconSize}
-                hideArrow
-                tooltip="SvelteKit"
-                icon="/docs/img/icons/svelte-icon"
-              />
-            </a>
-          </Link>
-          <Link href={`/guides/getting-started/quickstarts/solidjs`} passHref>
-            <a className="no-underline">
-              <IconPanel
-                iconSize={iconSize}
-                hideArrow
-                tooltip="SolidJS"
-                icon="/docs/img/icons/solidjs-icon"
-              />
-            </a>
-          </Link>
-          <Link href={`/guides/getting-started/quickstarts/vue`} passHref>
-            <a className="no-underline">
-              <IconPanel
-                iconSize={iconSize}
-                hideArrow
-                tooltip="Vue"
-                icon="/docs/img/icons/vuejs-icon"
-              />
-            </a>
+          {frameworks.map((framework) => (
+            <Link href={framework.href} passHref>
+              <a className="no-underline">
+                <IconPanel
+                  iconSize={iconSize}
+                  hideArrow
+                  tooltip={framework.tooltip}
+                  icon={framework.icon}
+                />
+              </a>
           </Link>
```

Comment:
We should probably add a `key` to these.

Reply:
Ps. Typically you want to try to avoid using index as key since a re-order will cause each child to re-render (not a big deal here as that wouldn't apply). Something unique like `href` would work.